gremlin-python/src/main/python/gremlin_python/driver/protocol.py

```diff
@@ -203,6 +60,12 @@ def __init__(self,
         self._message_serializer = message_serializer
         self._username = username
         self._password = password
+        self._response_msg = {'status': {'code': 0,
```

Comment:
0 is probably not the right default for code unless you are explicitly checking for 0 as a sentinel value.

Reply:
I think I'll leave it for now, these values do get replaced in the chunking methods as soon as we read the response.